### ISSUE TITLE
Missing `.zbin` files

### ISSUE THREAD

**Issue by imqdee** [lambdaclass/zksync_era_precompiles#225]
Hey guys,

First, thanks for all!
I cloned the main branch of the repository and then tried to run a node using `make run-node` or `make run-node-light`  without success.

Here is the errors

```bash
Compiling era_test_node v0.1.0-alpha.14 (/Users/qdqd/workspace/zksync_era_precompiles/.test-node-subtree)
error: couldn't read src/deps/contracts/EcAddG2.yul.zbin: No such file or directory (os error 2)
   --> src/deps/system_contracts.rs:168:13
    |
168 |             include_bytes!("contracts/EcAddG2.yul.zbin").to_vec(),
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: this error originates in the macro `include_bytes` (in Nightly builds, run with -Z macro-backtrace for more info)

error: couldn't read src/deps/contracts/EcMulG2.yul.zbin: No such file or directory (os error 2)
   --> src/deps/system_contracts.rs:178:13
    |
178 |             include_bytes!("contracts/EcMulG2.yul.zbin").to_vec(),
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: this error originates in the macro `include_bytes` (in Nightly builds, run with -Z macro-backtrace for more info)
    
error: could not compile `era_test_node` (lib) due to 2 previous errors
make[1]: *** [rust-build] Error 101
make: *** [/Users/XXXX/YYYYY/zksync_era_precompiles/.test-node-subtree/target/release/era_test_node] Error 2
```
    
It seems that both `EcAddG2.yul.zbin` and `EcMulG2.yul.zbin` are missing on main. Am I missing something? Should I generate them or something? I am not familiar with this file format.

Any advice would be appreciated.

**Comment by IAvecilla** [comment]
Hey @qd-qd! thanks for opening the issue can you provide the Rust version that you are using to compile this?

**Comment by rdubois-crypto** [comment]
Hello.

Working on macosx, with M3pro, rustc 1.78.0-nightly (0ecbd0605 2024-02-25), i face the same errors as described above.

**Comment by IAvecilla** [comment]
Okay, I see. It's possible that I have a strange local state of the precompiles. Let me get back to you with a more detailed answer.

**Comment by IAvecilla** [comment]
Hey @rdubois-crypto, @qd-qd, I believe the required files have been successfully pushed. Could you please attempt the compilation again to confirm if the issue has indeed been resolved? If not, please feel free to reopen the issue.

**Comment by rdubois-crypto** [comment]
#225 is fixed as well as #223, however a new issue occured with ecmul, described in #232
